src/OrleansAWSUtils/IServiceCollectionExtensions.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Options;
+using Orleans;
+using Orleans.Runtime.Configuration;
+using Orleans.Runtime.MembershipService;
+using OrleansAWSUtils.Configuration;
+
+namespace OrleansAWSUtils
+{
+    public static class IServiceCollectionExtensions
```

Comment:
whatever namespace we decide, all extensions will probably be in the same namespace, so make the name of this class be more specific, such as DynamoDbServiceCollectionExtensions

Reply:
I thin it should be like AWSUtilsServiceCollectionExtensions. So future extension methods can belong here too .

Reply:
it might be that the current assembly for AWS is not correctly split, but we intend to split some of them, so don't take that as a pattern for everything, even if it currently applies for this one.
In theory, I would expect a nuget with the minimum dependencies to use DynamoDb, which should be separate from the stuff related to SQS.
On a separate note, design guidelines for naming say that you should treat acronyms like 1 pascal cased word, so if you do want to go with that name, it would be `AwsUtilsServiceCollectionExtensions`. We probably don't follow this everywhere, but this is likely one of the few public types with that incorrect casing that the user sees anyway.

Reply:
hmm. we can rename this file when we did split them. No need to over-complicate the naming now. I will take AwsUtilsServiceCollectionExtensions